client/src/PublishTenderForm.js

```diff
@@ -0,0 +1,178 @@
+import React, { useState, useEffect } from "react";
+import "./styles.css";
+
+const PublishTenderForm = () => {
+	const [title, setTitle] = useState("");
+	const [description, setDescription] = useState("");
+	const [closingDate, setClosingDate] = useState("");
+	const [announcementDate, setAnnouncementDate] = useState("");
+	const [deadlineDate, setDeadlineDate] = useState("");
+	const [skills, setSkills] = useState([]);
+	const [selectedSkills, setSelectedSkills] = useState([]);
+
+	useEffect(() => {
+		fetch("/api/skills")
+			.then((response) => response.json())
+			.then((data) => {
+				setSkills(data.skills);
+			})
+			.catch((error) => {
+				console.error("Error fetching skills:", error);
```

Comment:
Logging to the console is not good enough. The user needs to know there was an error